packages/web/src/scss/components/Item/README.md

```diff
@@ -1,10 +1,88 @@
 # Item
 
-The Item component is used to display a single item in a list. Currently we support
-two types of item content: Radio and Checkbox. In order to unify and simplify
-API of these components and also to avoid repeating ourselves, we use Item as their modifier.
+The Item component is used to display a single item in a list. It can be used in Dropdown or similar.
```

Comment:
Please, add a link to the Dropdown as you have done it for the Checkbox.

Reply:
I wouldn't mention `Dropdown` at all until it's actually supported.

Reply:
I will add a comment to the issue where we actually use Item in the dropdown.